src/DataTable/DataTableLayout.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import SideFilters from './SideFilters';
+import DataTableContext from './DataTableContext';
+
+// handles layout for side of table
+
+const DataTableLayout = ({
+  className,
+  children,
+}) => {
+  const { setFilter, showFiltersInSidebar } = useContext(DataTableContext);
+
+  return (
+    <div className={classNames('pgn__data-table-layout-wrapper', className)}>
+      {/* <div ref={ref} className={classNames('pgn__data-table-layout-wrapper', className)}> */}
+      {(showFiltersInSidebar && setFilter) && (
+        <div className="pgn__data-table-layout-sidebar">
+          <SideFilters />
+        </div>
+      )}
+      <div className="pgn__data-table-layout-main">
+        {children}
+      </div>
+    </div>
+  );
+};
+
+DataTableLayout.defaultProps = {
+  className: null,
+  children: null,
+};
+
+DataTableLayout.propTypes = {
+  className: PropTypes.string,
+  children: PropTypes.node,
```

Comment:
Should `children` be a required prop?

Reply:
@manny-m Let's make `children` a required prop since we'll always expect `children` to be passed.